playbooks/roles/weave-network/files/weave-daemonset-k8s-v1.9.5.yaml

```diff
@@ -0,0 +1,118 @@
+kind: ClusterRole
```

Comment:
Can't we find it versioned on the internet? Or do they only provide unversioned Weave?